modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -325,7 +328,8 @@ class Editor implements EditorObservable {
     }
 
     this.ui = {
-      registry: registry()
+      registry: registry(),
+      styleSheetLoader: undefined
```

Comment:
Could this be defined when the ui is constructed but queue the things to be loaded until it gets attached to the dom or error out if someone tries to load before its attached it would prefer the former if possible.

Reply:
I could, but it incurs some code and complexity to handle this.

Untested experiment at branch `feature/TINY-6089-4`. @spocke @lnewson let me know if this is worth pursuing. I don't think it's worth it.

Reply:
We kind of already have a queue for content css style sheets the editor has a contentCss array that gets mutated when styles are loaded would have been nice to replace that with a loader and not needing to have to worry about when in the life cycle you add stylesheet it would just magically work but if it's to hard then I guess we shouldn't bother but it shouldn't be that complex to implement.

Reply:
I've logged TINY-6119 to address these concerns. The QueuedStyleSheetLoader in my spike should address both problems, but I think it's a out of scope for the current task.

Are you ok with delaying this improvement, and dealing with it in TINY-6119?

Reply:
Yeah this is scope creep but since you made a new loader it could be improved.